utils/rpms/daos.spec

```
@@ -3,7 +3,7 @@
 %define agent_svc_name daos_agent.service
 %define sysctl_script_name 10-daos_server.conf
 
-%global mercury_version 2.1.0~rc4-6%{?dist}
+%global mercury_version 2.1.0~rc4-7%{?dist}
```

The latest is 8 now, not 7.

```suggestion
%global mercury_version 2.1.0~rc4-8%{?dist}
```

---

All review comments have been addressed in the latest submission.